### PR TITLE
Minor: Fix kafkaconnect test connection

### PR DESCRIPTION
**Pull request by SumanMaharana**
### Describe your changes:

Minor: Fix kafkaconnect test connection

#
### Type of change:

- [x] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: ingestion/src/metadata/ingestion/source/pipeline/kafkaconnect/client.py [hunk 1]
```diff
@@ -61,15 +61,22 @@ def get_cluster_info(self) -> Optional[dict]:
...
         """
-        try:
-            result = self.client.get_cluster_info()
-            return result
-
-        except Exception as exc:
-            logger.debug(traceback.format_exc())
-            logger.error(f"Unable to get cluster info :{exc}")
+        result = self.client.get_cluster_info()
+        return result
```
Comment (pmbrull) [inline ingestion/src/metadata/ingestion/source/pipeline/kafkaconnect/client.py:65]: lets just return `self.client.get_cluster_info()`

File: ingestion/src/metadata/ingestion/source/pipeline/kafkaconnect/client.py [hunk 2]
```diff
@@ -61,15 +61,22 @@ def get_cluster_info(self) -> Optional[dict]:
...
+    def get_connectors_list(
+        self,
+        expand: str = None,
+        pattern: str = None,
+        state: str = None,
+    ) -> dict:
+        """
+        Get the list of connectors from Kafka Connect cluster.
+        """
+        result = self.client.list_connectors(
```
Comment (pmbrull) [inline ingestion/src/metadata/ingestion/source/pipeline/kafkaconnect/client.py:76]: same

**Review by pmbrull — CHANGES_REQUESTED** [review]

File: ingestion/src/metadata/ingestion/source/pipeline/kafkaconnect/client.py (hunk 1 above)
Comment (SumanMaharana) [reply ingestion/src/metadata/ingestion/source/pipeline/kafkaconnect/client.py]: done

File: ingestion/src/metadata/ingestion/source/pipeline/kafkaconnect/client.py (hunk 2 above)
Comment (SumanMaharana) [reply ingestion/src/metadata/ingestion/source/pipeline/kafkaconnect/client.py]: done

**Review by pmbrull — APPROVED** [review]

**Pull request merged**